### PR TITLE
Data contribution 2

### PR DESCRIPTION
**Pull request by nafeldman**
Hi Education Team! Please review as your time allows. Our goal is to have these ready to be published by the end of the month.

### CODE REVIEW

File: data_contribution_2_data.md [hunk 1]
```diff
@@ -0,0 +1,907 @@
...
+author:   Arcus Library Sciences
+email: dlarcuslibraryscience@chop.edu
+version:  1.0.0
+current_version_description: Brief description of why this version exists
+module_type: standard
+docs_version: 1.1.0
+language: en
+narrator: UK English Female
+title: Arcus Project Template Orientation
+comment:  Arcus Archives serves as the canonical repository for research data at CHOP. Archiving data with Arcus safeguards and maintains research data in accordance with established archival protocols, also while promoting data sharing. The Data Contribution Module provides an overview of the reasons to archive research data with Arcus, as well as the processes involved in scoping data, conducting privacy assessments, and addressing technical requirements prior to data submission. This Module explores the Project Template directory structure, and delves into the specific steps required for archiving data within the Arcus platform.
```
Comment (pm0kjp) [inline data_contribution_2_data.md:12]:
```suggestion
comment: Learn about how to disclose data to CHOP by making a data contribution!
long_description:  Arcus Archives serves as the canonical repository for research data at CHOP. Archiving data with Arcus safeguards and maintains research data in accordance with established archival protocols, also while promoting data sharing. The Data Contribution Module provides an overview of the reasons to archive research data with Arcus, as well as the processes involved in scoping data, conducting privacy assessments, and addressing technical requirements prior to data submission. This Module explores the Project Template directory structure, and delves into the specific steps required for archiving data within the Arcus platform.
```

Note that you're missing a "long_description" so I've taken the liberty of adding a comment and making your comment the long_description.  Feel free to tweak, this is just my suggestion.

File: data_contribution_2_data.md [hunk 2]
```diff
@@ -0,0 +1,907 @@
...
+
+### Future data contributors
+
+This module is an overview of CHOP's Project Template structure, all contributed data is arranged in this structure for archiving in Arcus. The Project Template is useful at all stages of research, we suggest implementing it as early as possible in your research as it provides a shared, documented framework for organizing a research effort.
+
+Arcus's Library Science team is happy to meet with researchers at all phases of research for research data management consultations and planning for future archival contributions. This includes early in your research project, as we can help set up a project template file directory structure for storing data and recommend metadata and organization best practices. In addition to this module, there are additional data management resources available on CHOP’s [Arcus resources page](https://www.research.chop.edu/applications/arcus/resources).
+
+If after viewing this module, you are prepared to archive data with Arcus, please fill out [the following request](https://pm.arcus.chop.edu/servicedesk/customer/portal/6/create/256) to start the process.
```
Comment (pm0kjp) [inline data_contribution_2_data.md:110]:
```suggestion
If after viewing this module, you are prepared to archive data with Arcus, please fill out [a data contribution request](https://pm.arcus.chop.edu/servicedesk/customer/portal/6/create/256) to start the process.
```
When I saw the word "following" I was looking for another line...

File: data_contribution_2_data.md [hunk 3]
```diff
@@ -0,0 +1,907 @@
...
+
+When appropriate, archiving your research in Arcus is expected from Scientific Projects with an Arcus Lab. This is documented in the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-use). Archiving is required if you would like to move any data created within an Arcus Lab to a new Scientific Project or if other research teams would like to reuse your data.
+
+When you are ready to archive your lab data, please submit the following request in the [Arcus Help Center](https://pm.arcus.chop.edu/servicedesk/customer/portal/6/create/256) to begin the data contribution process.
+
+## Arcus Data Lifecycle
+
+Arcus' goal for research data management and the project template is to provide tools that are relevant throughout the entire lifecycle of research data. The project template is designed to be adaptable and iterative to capture the wide range of research activities at CHOP. The project template combines flexibility to encompass diverse data capturing needs, while also maintaining a consist structure for all archived data which facilitates seamless communication among various projects spanning different domains, thereby promoting effective data sharing.
```
Comment (pm0kjp) [inline data_contribution_2_data.md:124]:
```suggestion
Arcus's goal for research data management and the project template is to provide tools that are relevant throughout the entire lifecycle of research data. The project template is designed to be adaptable and iterative to capture the wide range of research activities at CHOP. The project template combines flexibility to encompass diverse data capturing needs, while also maintaining a consist structure for all archived data which facilitates seamless communication among various projects spanning different domains, thereby promoting effective data sharing.
```
I *think* the current "possessive apostrophe following S" rule includes another s?  But feel free to disregard, I haven't actually researched this.

File: data_contribution_2_data.md [hunk 4]
```diff
@@ -0,0 +1,907 @@
...
+- Microbiome center
+- Research IT
+- Clinical sequening unit
+- Medical Informatics Unit
+
+**How is the Project Template used at Arcus?**
+
+The Project Template prioritizes streamlined archiving and reproducible research pathways. It archives a wide range of research types from the Research Institute, making them discoverable through tools like [Arcus Cohort Discovery](https://arcus.chop.edu), [Gene](https://chop.alationcloud.com), and [Omics Variant Browser](https://arcus.chop.edu). The Project Template facilitates organizing diverse research data in a single directory structure, enabling automated archiving, metadata management, and data delivery throughout the research data lifecycle. This file directory structure is used for the entire lifecycle of research data within Arcus:
+
+![Graphic representation of the lifecycle of the project template in Arcus](media/project_template/ProjectTemplate_Lifecycle.png)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:143]: Ideally any text on this image (and other images that similarly convey information via text) is captured in alt text or other text elsewhere in the document.  This is important for accessibility, mobile viewing where an image might be v small on a phone, as well as search.  But don't let this hold you back, we can't let the perfect be the enemy of the good!

File: data_contribution_2_data.md [hunk 5]
```diff
@@ -0,0 +1,907 @@
...
+- **_Manifests_** (data)\*
+- **_Models_** (access tools)
+- **_References_** (contextual)
+- **_Reports_** (contextual)
+- **_Requirements_** (contextual)
+- **_SRC_** (access tools)
+
+Below is an image of the entire Project Template Directory, with more detail about each section:
+
+![Graphic respresentation of the Project template with a short explanation of all the different sections](media/project_template/ProjectTemplate_Description.png)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:166]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases.  I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 6]
```diff
@@ -0,0 +1,907 @@
...
+
+![Graphic representation of the lifecycle of the project template in Arcus](media/project_template/ProjectTemplate_Lifecycle.png)
+
+The project template provides a shared structure so that institutional knowledge previously held locally by various members of the data creation team becomes centralized.
+
+The utility of [the project template](https://github.research.chop.edu/arcus/rdm-project-template) for lab drive organization and integration with the Arcus archives is summarized in the graphic below.
+
+![Graphic representation of the workflow of research data and the project tempalte](media/project_template/ProjectTemplate_workflow.png)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:149]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 7]
```diff
@@ -0,0 +1,907 @@
...
+
+Below is an image of the entire Project Template Directory, with more detail about each section:
+
+![Graphic respresentation of the Project template with a short explanation of all the different sections](media/project_template/ProjectTemplate_Description.png)
+
+### Research Data
+
+The Project Template brings together three categories of information: Research Data, Access Tools and Contextual files. Research data is the actual data collected during the course of research processes used for analysis. The manifests describe this data, crosswalking files to participants. Research data (with manifests) is the mininum required information for all Arcus data contributions.
+
+![Graphic respresentation of the Project template with that denotes research data sections](media/project_template/research_data.png)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:172]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 8]
```diff
@@ -0,0 +1,907 @@
...
+
+### Access Tools
+
+Access Tools are the code used to do the analysis. This can include machine models, scripts and Jupyter notebooks.
+
+![Graphic respresentation of the Project template with that denotes access tool sections](media/project_template/access_tools.png)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:178]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 9]
```diff
@@ -0,0 +1,907 @@
...
+
+### Contextual Files
+
+Contextual Files provide information needed to understand the data and analysis. This can include omics protocols, data dictionaries, reports and diagrams.
+
+![Graphic respresentation of the Project template with that denotes contextual files sections](media/project_template/contextual_files.png)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:184]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 10]
```diff
@@ -0,0 +1,907 @@
...
+- Bioinformatics
+- Cancer research
+- Microbiome center
+- Research IT
+- Clinical sequening unit
+- Medical Informatics Unit
+
+**How is the Project Template used at Arcus?**
+
+The Project Template prioritizes streamlined archiving and reproducible research pathways. It archives a wide range of research types from the Research Institute, making them discoverable through tools like [Arcus Cohort Discovery](https://arcus.chop.edu), [Gene](https://chop.alationcloud.com), and [Omics Variant Browser](https://arcus.chop.edu). The Project Template facilitates organizing diverse research data in a single directory structure, enabling automated archiving, metadata management, and data delivery throughout the research data lifecycle. This file directory structure is used for the entire lifecycle of research data within Arcus:
```
Comment (pm0kjp) [inline data_contribution_2_data.md:141]: These links don't actually take you into the applications, so consider something like:

```suggestion
The Project Template prioritizes streamlined archiving and reproducible research pathways. It archives a wide range of research types from the Research Institute, making them discoverable through tools like Arcus Cohort Discovery, available as an application [on the Arcus website](https://arcus.chop.edu), [Gene](https://chop.alationcloud.com), and Omics Variant Browser, [available as an application [on the Arcus website](https://arcus.chop.edu). The Project Template facilitates organizing diverse research data in a single directory structure, enabling automated archiving, metadata management, and data delivery throughout the research data lifecycle. This file directory structure is used for the entire lifecycle of research data within Arcus:
```

File: data_contribution_2_data.md [hunk 11]
```diff
@@ -0,0 +1,907 @@
...
+
+- Arcus delivered archival data will be found here.
+- Study team generated data brought into Arcus goes here.
+- This data is managed by Arcus, and should not be modified by the research team.
+
+**Raw** data is different depending on the type of research. Please select what type(s) of data you would like more information about, you can select both:
+
+- [ ] omics data
+- [ ] clinical data
+<script output="data_type">"@input"</script>
```
Comment (pm0kjp) [inline data_contribution_2_data.md:235]: Clicking these, I'm left with a big empty space afterwards that looks like nothing happened unless I scroll down the page.  Consider checking the functions?  I think there may be a missing image or one that is being hidden?

File: data_contribution_2_data.md [hunk 12]
```diff
@@ -0,0 +1,907 @@
...
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type`)
+
+  if(data_type[0]) {
+    send.liascript(`## Omics Data
+
+
+![Omics data/raw](media/project_template/raw_omics.gif)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:245]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 13]
```diff
@@ -0,0 +1,907 @@
...
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type`)
+
+  if(data_type[1]) {
+    send.liascript(`## Clinical Data
+
+
+![Clinical data/raw](media/project_template/raw_clinical.gif)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:266]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 14]
```diff
@@ -0,0 +1,907 @@
...
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_interim`)
+
+  if(data_type[0]) {
+    send.liascript(`
+    ## Omics Data
+
+![Omics data/interim](media/project_template/interim_omics.gif)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:324]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 15]
```diff
@@ -0,0 +1,907 @@
...
+</script>
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_interim`)
+
+  if(data_type[1]) {
+    send.liascript(`
+    ## Clinical Data
+![Clinical data/interim](media/project_template/interim_clinical.gif)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:341]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 16]
```diff
@@ -0,0 +1,907 @@
...
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_endpoints`)
+
+  if(data_type[0]) {
+    send.liascript(`
+    ## Omics Data
+
+![Omics data/endpoints](media/project_template/endpoints_omics.gif)<!-- style = "max-height: 500px;" -->
```
Comment (pm0kjp) [inline data_contribution_2_data.md:372]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 17]
```diff
@@ -0,0 +1,907 @@
...
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_endpoints`)
+
+  if(data_type[1]) {
+    send.liascript(`
+    ## Clinical Data
+
+![Clinical data/endpoints](media/project_template/endpoints_clinical.gif)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:392]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 18]
```diff
@@ -0,0 +1,907 @@
...
+
+Manifests are an inventory of all data in the collection, and provide a mapping between research data in the data folders, and participant information. The manifests also create a mapping between data and associated pipeline and technical information about workflows. There are three main manifests that are mandatory for every archival collection:
+
+- file_manifest.csv
+- participant_manifest.csv
+- partcipant-crosswalk.txt
+
+Additional manifests are only required if needed for the data or collection type. These files are detailed in the next sections. The graphic below illustrates the linking between the files:
+
+![ID Crosswalks between manifests](media/project_template/manifests_linkages.png)
```
Comment (pm0kjp) [inline data_contribution_2_data.md:465]: Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.

File: data_contribution_2_data.md [hunk 19]
```diff
@@ -0,0 +1,907 @@
...
+
+**Within an [Arcus Scientific Lab](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/main/arcus_orientation.md#1)**
+
+- Managed by Arcus, you will not need to create these for yourselves
+- This will only appear in the lab if archival data is delivered
+
+### manifests/file_manifest
+
+The file_manifest.csv matches the biosample_id to each file in the data folders. Below is more detail about each section in the file:
```
Comment (pm0kjp) [inline data_contribution_2_data.md:474]:
```suggestion
The file\_manifest.csv matches the biosample\_id to each file in the data folders. Below is more detail about each section in the file:
```
If you don't escape underscores, markdown thinks you want the enclosed text italicized if there are two underscores on the same line.

File: data_contribution_2_data.md [hunk 20]
```diff
@@ -0,0 +1,907 @@
...
+
+- biosample_id is an ID number for each file. For some studies, each file is derived about specific biosamples, so we suggest using the sample id. ideally, biosample_id links to the CHOP biobank. When you cannot link the the biobank, treat biosample_id as the IDs you use for samples taken from participants. For studies where there are no biosamples, the biosample_id can be the file name.
```
Comment (pm0kjp) [inline data_contribution_2_data.md:476]:
```suggestion
- biosample\_id is an ID number for each file. For some studies, each file is derived about specific biosamples, so we suggest using the sample id. ideally, biosample\_id links to the CHOP biobank. When you cannot link the the biobank, treat biosample\_id as the IDs you use for samples taken from participants. For studies where there are no biosamples, the biosample\_id can be the file name.
```

File: data_contribution_2_data.md [hunk 21]
```diff
@@ -0,0 +1,907 @@
...
+- file_type is the type of file, indicated by the file extension
+- protocol is only for omics data, select the omics data example below for more information
+- file_path is the file path for each file in the **_data\_** folders. File paths should start with **_data\_** and end with the full file name with extension
```
Comment (pm0kjp) [inline data_contribution_2_data.md:479]:
```suggestion
- file\_path is the file path for each file in the **\_data\_** folders. File paths should start with **\_data\_** and end with the full file name with extension
```

File: data_contribution_2_data.md [hunk 22]
```diff
@@ -0,0 +1,907 @@
...
+<script output="data_type_fm">"@input"</script>
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_fm`)
+
+  if(data_type[0]) {
+    send.liascript(`## Omics Data
+
+For Omics data, the **file_manifest.csv** matches biosample IDs to data files and experimental protocols, described in _yaml_ files. More information about the protocols files is available in the references section of this module. Many files might share the same experimental protocol. These yaml protocol files describe experiment and data processing details. See below for the columns in a file_manifest:
```
Comment (pm0kjp) [inline data_contribution_2_data.md:494]:
```suggestion
For Omics data, the **file\_manifest.csv** matches biosample IDs to data files and experimental protocols, described in \_yaml\_ files. More information about the protocols files is available in the references section of this module. Many files might share the same experimental protocol. These yaml protocol files describe experiment and data processing details. See below for the columns in a file\_manifest:
```

File: data_contribution_2_data.md [hunk 23]
```diff
@@ -0,0 +1,907 @@
...
+| file_path               | Use one file path per row. It should start with data/.                                                                                                                                                                                                                                                                           | String |
+| file_groups             | Files in the same group are related. Paired fastq files belong in the same group. A bam file and its index belong in the same group. Plink bfiles belong in the same group.                                                                                                                                                      | String |
+| derived\_from\_file\_group | This column describes relations between file groups. We want to capture consecutive pipeline steps. For example, a bam file is derived from a paired fastq group. Use the name of the file_groups used to construct this file. Delimit multiple groups with a semicolon. Use NA when there are no prior step files to reference. | String |
```
Comment (pm0kjp) [inline data_contribution_2_data.md:503]:
```suggestion
| derived\_from\_file\_group | This column describes relations between file groups. We want to capture consecutive pipeline steps. For example, a bam file is derived from a paired fastq group. Use the name of the file_groups used to construct this file. Delimit multiple groups with a semicolon. Use NA when there are no prior step files to reference. | String |
```
Strangely this is still showing up with weird italicizing for me, even though you're escaping.  Not sure what's up?

File: data_contribution_2_data.md [hunk 24]
```diff
@@ -0,0 +1,907 @@
...
+    `)
+  } else send.clear()
+} catch(e) { }
+</script>
+
+### manifests/participant_manifest
+
+The participant_manifest.csv identifies which participants information links to each of the files in the file_manifest. Below is more detail about each section of the file:
+
+- local_participant_id is a local identifier the study team used to identify the patient
```
Comment (pm0kjp) [inline data_contribution_2_data.md:536]:
```suggestion
- local\_participant\_id is a local identifier the study team used to identify the patient
```

File: data_contribution_2_data.md [hunk 25]
```diff
@@ -0,0 +1,907 @@
...
+<script output="data_type_pm">"@input"</script>
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_pm`)
+
+  if(data_type[0]) {
+    send.liascript(`## Omics Data
+
+The **participant_manifest.csv** matches participants (or patients) to cohorts and biosample IDs from the **file_manifest.csv**. Ideally, biosample\_id links to the CHOP biobank. When you cannot link to the biobank, treat biosample\_id as the IDs you use for samples taken from participants. If you deal with only one sample type, you might use the participant\_id. If you run a treatment/control experiment, you might use participant\_id and treat participant\_id\_control as as a biosample ID scheme. If you work with different tissue samples from participants, you might use participant\_id\_tissue as a biosample ID scheme.
```
Comment (pm0kjp) [inline data_contribution_2_data.md:553]: This is still showing up weirdly italicized for me?  Again, not sure what's up

File: data_contribution_2_data.md [hunk 26]
```diff
@@ -0,0 +1,907 @@
...
+
+| column               | definition                                                                                                                    | type   |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------- | ------ |
+| local\_participant\_id | This ID uniquely defined a person, and can be linked to an MRN.                                                               | String |
```
Comment (pm0kjp) [inline data_contribution_2_data.md:557]: This is still showing up weirdly italicized for me? Again, not sure what's up

File: data_contribution_2_data.md [hunk 27]
```diff
@@ -0,0 +1,907 @@
...
+| cohort               | Use this column to group participants into cohorts that will be cwopared (For example, case vs healthy control).              | String |
+| biosample_id         | Ideally, this ID can link to the CHOP biobank. When this is not possible, use the sample ID from your project.                | String |
+| family_id            | When participants are related, use family_id to group related participants. With trios or duos, the proband ID is often used. | String |
+| family_role          | Use a term from [eHB\_relationship\_types\_as\_of\_10\_30.json](https://github.research.chop.edu/arcus/rdm-project-template/blob/master/manifests/data_dicts/eHB_relationship_types_as_of_10_30.json) to indicate mother, father, proband, sister, etc..                    | String |
```
Comment (pm0kjp) [inline data_contribution_2_data.md:561]: This is still showing up weirdly italicized for me? Again, not sure what's up

File: data_contribution_2_data.md [hunk 28]
```diff
@@ -0,0 +1,907 @@
...
+| -------------------- | --------------------------------------------------------------------------------------------------------------------- | ------ |
+| local\_participant\_id | The local id of a participant.                                                                                        | String |
+| local\_relative\_id    | The local id of a relative to the participant.                                                                        | String |
+| relative\_family\_role | The familial relationship of the relative to the participant. Use terms from eHB\_relationship\_types\_as_of\_10\_30.json. | String |
+
+### manifests/familyid_crosswalk
+
+- This manifest should be used with trio and cohort omics data. A trio will contain three participants, a cohort can contain hundreds. This file walks the name for the trio or cohort file with the local_participant_id's included in it.
+- This is also called a PED or pedigree file in bioinformatics workflows. A pedigree is a structured description of the familial relationships between samples, see this [link for more information](https://gatk.broadinstitute.org/hc/en-us/articles/360035531972-PED-Pedigree-format). 
+
```
Comment (pm0kjp) [inline data_contribution_2_data.md:622]:
```suggestion

<!-- data-type="none" -->
```
Do you want the graph type "parallel crosswalk" to appear?  If not, I suggest you add the data type to the top of the table.

File: data_contribution_2_data.md [hunk 29]
```diff
@@ -0,0 +1,907 @@
...
+### manifests/file_derivation.csv
+
+The **file_derivation.csv** manifest is only required for omics contributions with multiple filetypes generated through a bioinformatics pipeline or workflow.
+
+**file_derivation.csv** describes the relationships between files in a pipeline or workflow.
+
+| column                 | definition                                                      | type   |
+| ---------------------- | --------------------------------------------------------------- | ------ |
+| destination\_file\_group | The files in this file group is derived from source_file_group. | String |
```
Comment (pm0kjp) [inline data_contribution_2_data.md:656]:
```suggestion
| destination\_file\_group | The files in this file group is derived from source\_file\_group. | String |
```

File: data_contribution_2_data.md [hunk 30]
```diff
@@ -0,0 +1,907 @@
...
+| source\_fil\e_group      | File group used to derive the destination_file_group.           | String |
```
Comment (pm0kjp) [inline data_contribution_2_data.md:657]:
```suggestion
| source\_file\_group      | File group used to derive the destination\_file\_group.           | String |
```

File: data_contribution_2_data.md [hunk 31]
```diff
@@ -0,0 +1,907 @@
...
+
+### manifests/env manifest
+
+For each script/notebook in _src/_, and each model in _models/_, there should be an env file (here env refers to a file named env with any extension, so env.yaml or env.txt, for example) that describes the environment in which it was created or run. Environment files should be named as follows: **descriptiveName_env** and placed in a folder called environments within the _configs/environment_ directory. Either individuals files or entire folders (whichever is the appropriate level) in scripts and notebooks within the _src/_ directory, or the _models/_ directory will need to be added to the env manifest file, matching them with their related environment file. See the see below for more infromation about this file:
```
Comment (pm0kjp) [inline data_contribution_2_data.md:661]:
```suggestion
For each script/notebook in \_src/\_, and each model in \_models/\_, there should be an env file (here env refers to a file named env with any extension, so env.yaml or env.txt, for example) that describes the environment in which it was created or run. Environment files should be named as follows: **descriptiveName\_env** and placed in a folder called environments within the \_configs/environment\_ directory. Either individuals files or entire folders (whichever is the appropriate level) in scripts and notebooks within the \_src/\_ directory, or the \_models/\_ directory will need to be added to the env manifest file, matching them with their related environment file. See the see below for more information about this file:
``` 

Again, I think I'm interpreting what you're doing... that you want these underscores to appear, right?  If you have two or more in a single line of text, you **must** prefix them with the backslash to escape their normal markdown usage (italics)

File: data_contribution_2_data.md [hunk 32]
```diff
@@ -0,0 +1,907 @@
...
+| related_environment   | Enter the environment filename. Some environment files will be entered multiple times as they relate to multiple files.                                                                                                                                                             | String |
+|                       |
+
+## src/
+
+The src or sources folder stores the access tools required to work with the research data and repeat the analysis. The need for access tools is dependent on the type of research, not all research has rules, scripts or notebooks. Any scripts saved in the src folder require an environment manifest to document the computing environment the code is run in, see the [environment manifest page](#manifestsenv_manifestcsv) for more information. Subdirectory folders can be customized and added as needed, below are the common directories or data types used in scientific research:
```
Comment (pm0kjp) [inline data_contribution_2_data.md:671]: Link doesn't go where you want it to, I think?

**Review by pm0kjp — CHANGES_REQUESTED** [review]
This looks good but I realized at around line 600 that I keep repeating the same things so I wanted to stop to give you a chance to look at what I've commented so far before I keep copy/pasting.  The basic things I keep repeating:

* Using text on an image to drive understanding is suboptimal for various reasons, but I can imagine that adding text into the document itself might be a huge PITA and worth doing as a "secondary improvement", so I won't hinder you moving forward on this
* There are lots of unintentional italics where you have 2 or more underscores on the same line of text without escaping them.

Take a look at what I've suggested (and continue looking especially for unintentional italics further on, I stopped noting them but you can see your script *in situ* at https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/data-contribution-2/data_contribution_2_data.md#1 to see what it looks like)

**Comment by nafeldman** [comment]
thank you @pm0kjp - that's a really good point about adding alt text (especially to the gifs) we should definitely go back and do that.

File: data_contribution_2_data.md (hunk 4 above)
Comment (allrolsen) [reply data_contribution_2_data.md]: Thank you for pointing this out Joy! We reused graphics we already has for this module, which is why there are so many images with text. We are not going to be able to lay these images out in CSS to match W3 standards. I went through and at least improved all the alt text. We will keep in mind these alt text standards in future.

File: data_contribution_2_data.md (hunk 31 above)
Comment (allrolsen) [reply data_contribution_2_data.md:662]: Thank you for catching these. Nicole and I thought we had corrected all of them, but more keep on showing up!

File: data_contribution_2_data.md (hunk 28 above)
Comment (allrolsen) [reply data_contribution_2_data.md:623]: We did not want a 'parallel crosswalk' and did not know how to get rid of it!

File: data_contribution_2_data.md (hunk 32 above)
Comment (allrolsen) [reply data_contribution_2_data.md]: Thank you, I removed the link

**Comment by allrolsen** [comment]
> This looks good but I realized at around line 600 that I keep repeating the same things so I wanted to stop to give you a chance to look at what I've commented so far before I keep copy/pasting. The basic things I keep repeating:
> 
> * Using text on an image to drive understanding is suboptimal for various reasons, but I can imagine that adding text into the document itself might be a huge PITA and worth doing as a "secondary improvement", so I won't hinder you moving forward on this
> * There are lots of unintentional italics where you have 2 or more underscores on the same line of text without escaping them.
> 
> Take a look at what I've suggested (and continue looking especially for unintentional italics further on, I stopped noting them but you can see your script _in situ_ at https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/data-contribution-2/data_contribution_2_data.md#1 to see what it looks like)

@pm0kjp Thank you for all of your edits and suggestions. I have gone through the text and ensured that all of the _ are properly escape. Despite this, some text is still italicizing. All the text that still does this is with the scripts. Is there someone we can reach out to at LiaScript to look at this?

**Comment by rosemm** [comment]
Hi @allrolsen I noticed the problem you pointed out about the escaped underscores not working within the contingent text. It's because that text is piped in as part of a script -- anything that needs to be escaped in that text has to be escaped twice (e.g. `\\_`) because one "layer" of the escaping happens as the text is parsed by the script, and you still need an escape present there when it comes out the other side and gets inserted into the page. 
I went through and put those extra escapes in for you, and I think it's all rendering correctly now --- let me know if you still see problems!

**Comment by rosemm** [comment]
I also saw a few typos. Do you have spell check enabled in your text editor? (https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) I'm a terrible speller, so I rely on it a lot :) It saves me a ton of time and errors.

**Comment by allrolsen** [comment]
> Hi @allrolsen I noticed the problem you pointed out about the escaped underscores not working within the contingent text. It's because that text is piped in as part of a script -- anything that needs to be escaped in that text has to be escaped twice (e.g. `\\_`) because one "layer" of the escaping happens as the text is parsed by the script, and you still need an escape present there when it comes out the other side and gets inserted into the page.
> I went through and put those extra escapes in for you, and I think it's all rendering correctly now --- let me know if you still see problems!

Thank you!! Nicole and I were really annoyed by this, and had no idea how to fix it.

**Comment by allrolsen** [comment]
> I also saw a few typos. Do you have spell check enabled in your text editor? (https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) I'm a terrible speller, so I rely on it a lot :) It saves me a ton of time and errors.

I did not have spell check enabled, thank you for sharing that tool. I added it to VS Code and fixed all the spelling mistakes.

**Comment by allrolsen** [comment]
@pm0kjp and @rosemm Thank you both again for reviewing this module, your suggestions and help are invaluable! @nafeldman and I fell it looks good and are ready to merge this branch. I am not able to do it, it says there are changes requested (believe I resolved all of them) and that there is a conflict (media/source/.DS_Store file). Could one of you merge it? Someone with an Admin role should be over to override these issues.

**Review by rosemm — APPROVED** [review]

**Comment by rosemm** [comment]
I fixed the merge conflict, so it should be ready to go now. 
Unfortunately, it looks like github won't let me merge it without @pm0kjp specifically signing off, since she had requested changes earlier. Joy's out today, so it might be a little while before she has a chance to approve.

**Review by pm0kjp — APPROVED** [review]

**Pull request merged**